ui/permissions_test.py

```diff
@@ -191,24 +191,11 @@ def test_is_collection_owner_user_superuser(collection_owner_permission, collect
 def test_is_collection_admin_anonymous_users(collection_permission, collection, request_data_anon):
     """
     Test for HasCollectionPermissions.has_object_permission to verify that
-    anonymous users do not have permission.
+    anonymous users have permission.
```

Comment:
it would be nice to add test where user change permission from private to public and assert that user other user can view